pom.xml

```diff
@@ -7,40 +7,117 @@
     <groupId>org.monarchinitiative.biodownload</groupId>
     <artifactId>biodownload</artifactId>
     <version>1.0-SNAPSHOT</version>
-    <name>biodownload</name>
+    <name>BioDownload</name>
+    <description>BioDownload is a small utility library used at the Robinson lab to download HPO, MONDO, UBERON, GO ontologies.</description>
     <url>https://github.com/TheJacksonLaboratory/biodownload</url>
+    <inceptionYear>2021</inceptionYear>
+
+    <licenses>
+        <license>
+            <name>3-clause BSD</name>
+            <url>http://www.opensource.org/licenses/bsd-license.php</url>
+            <distribution>repo</distribution>
+        </license>
+    </licenses>
+
+    <developers>
+        <developer>
+            <name>Baha El Kassaby</name>
+            <email>baha.elkassaby@jax.org</email>
```

Comment:
@belkassaby is it OK to include your credentials here in this way?

Reply:
sure